include/z64math.h

```diff
@@ -7,6 +7,11 @@
 #define SQ(x) ((x)*(x))
 #define VEC_SET(V,X,Y,Z) (V).x=(X);(V).y=(Y);(V).z=(Z)
 
+typedef union FloatInt {
+    f32 f;
+    s32 i;
```

Comment:
Why `s32` instead of `u32`?
I think unsigned makes more sense since this is used to do bitwise operations.

Reply:
good point, that would make fault.c easier in the future too